packages/terra-site/webpack.config.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
           options: {
             sourceMap: true,
             importLoaders: 2,
-            localIdentName: '[path]___[name]__[local]___[hash:base64:5]',
```

Comment:
Im not sure of your question. The current className output is: 
`_-terra-button-lib-___Button__button___1303G`

this config changes it to:

`Button__button___1303G`